docs/Clearwater_Configuration_Options_Reference.md

```diff
@@ -265,6 +265,26 @@ This section describes the static DNS config which can be used to override DNS r
       ]
     }
 
+## RPH Config
+
+This section describes how to configure the priorities that should be given to different Resource Priority Header values. These options should be set in a local copy by running `cw-config download rph_json`, editing this downloaded copy and then running `cw-config upload rph_json` when finished. Currently, only the Namespaces and Priority-Valuess mentioned in RFC 4412 are supported as Resource Priority Header values. This file has the format:
+
```

Comment:
I think you want the example JSON to be in a code block. You can do that by putting "```" around it.